srcpkgs/praat/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'praat'
 pkgname=praat
-version=6.2.10
+version=6.2.11
 revision=1
 create_wrksrc=yes
 hostmakedepends="pkg-config"
 makedepends="gtk+3-devel $(vopt_if alsa alsa-lib-devel) $(vopt_if jack jack-devel)
  $(vopt_if pulseaudio 'alsa-lib-devel pulseaudio-devel')"
+checkdepends="liberation-fonts-ttf"
```

Comment:
```suggestion
checkdepends="cantarell-fonts"
```
cantarell-fonts are much smaller and we use them in other packages for tests

Reply:
Thanks for the suggestion, did a quick test locally and all works out. :)